src/slot_migrate.cc

```diff
@@ -104,10 +104,9 @@ Status SlotMigrate::MigrateStart(Server *svr, const std::string &node_id, const
   dst_node_ = node_id;
 
   // Create migration job
-  SlotMigrateJob *job = new SlotMigrateJob(slot, dst_ip, dst_port,
-                                           speed, pipeline_size, seq_gap);
+  slot_job_ = std::unique_ptr<SlotMigrateJob>(new SlotMigrateJob(slot, dst_ip, dst_port,
+                                           speed, pipeline_size, seq_gap));
   std::lock_guard<std::mutex> guard(job_mutex_);
```

Comment:
(not related to changes of this PR) It is weird since seems the scope of this `lock_guard` includes a `LOG(INFO)`, which has IO operations.